dnsmasq.template

```diff
@@ -0,0 +1,25 @@
+# Template file for dnsmasq generation
+# dnsmasq will default to sending v4 + v6 address of dnsmasq server
+
+# Enforce some standards
+bogus-priv
+domain-needed
+
+# Defaults on the pi
```

Comment:
I feel like people will use `dnsmasq` more than just on a raspberry pi, so pi-specific stuff maybe shouldn't go here?

Reply:
They can delete it. Will comment better.